lib/periodic_jobs.rb

```diff
@@ -150,5 +150,8 @@
   # Notifies slack channel if certain benefits states get stuck
   mgr.register('0 2,9,16 * * 1-5', 'VBADocuments::FlipperStatusAlert')
   # Checks status of Flipper features expected to be enabled and alerts to Slack if any are not enabled
+
+  # Rotates Lockbox/KMS record keys and ciphertexts every October 12th (when the KMS key auto-rotate)
+  mgr.register('0 0 12 10 *', 'KmsKeyRotation::BatchInitiatorJob')
```

Comment:
I wonder if we should start this later in the day? This is set to run at 9pm Pacific.